tests/components/upnp/test_config_flow.py

```diff
@@ -327,7 +327,8 @@ async def test_options_flow(hass: HomeAssistantType):
         await hass.async_block_till_done()
 
         # DataUpdateCoordinator gets a default of 30 seconds for updates.
-        coordinator = hass.data[DOMAIN][DOMAIN_COORDINATORS][mock_device.udn]
+        device = hass.data[DOMAIN][DOMAIN_DEVICES][mock_device.udn]
```

Comment:
If we want to test the update interval we should move time forward and check that the update happens at the correct interval.

Use this:
https://github.com/home-assistant/core/blob/de4b1eebdd6826658574c53d0026d1db1c6cfb66/tests/common.py#L363-L365

We don't want to access integration details, like hass.data or the coordinator, in the tests.